### PR TITLE
[Perf] CO CICT App Workflows

### PR DESCRIPTION
**Pull request by kiran-tripathy**
## Summary

Automation of the manual performance tracking for various workflows of COCICT app.

### Link to Jira Ticket (if applicable)
[https://dimagi-dev.atlassian.net/browse/QA-4385](https://dimagi-dev.atlassian.net/browse/QA-4385)

## QA Checklist

- [x] Label(s) and Assignee added
- [x] PR sent for review
- [x] Documentation (if applicable) added/updated

### CODE REVIEW

File: common_utilities/generate_output_path.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import os
+
+from common_utilities.path_settings import PathSettings
+
+
+def generate_output_path():
+    OUTPUT_PATH = "OutputFiles/"
+    output_path = (os.path.abspath(os.path.join(PathSettings.BASE_DIR, OUTPUT_PATH)))
+    print('Path not present')
```
Comment (kbo001) [inline common_utilities/generate_output_path.py:9]: should this print statement be inside the if block?

File: common_utilities/path_settings.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ class PathSettings:
     else:
         DOWNLOAD_PATH = Path('~/Downloads').expanduser()
 
-    NY_ROOT = os.path.abspath(os.path.join(BASE_DIR, os.pardir, os.pardir))
+    #NY_ROOT = os.path.abspath(os.path.join(BASE_DIR, os.pardir, os.pardir))
```
Comment (kbo001) [inline common_utilities/path_settings.py:17]: you can remove this if not required

**Review by kbo001 — CHANGES_REQUESTED** [review]

File: common_utilities/path_settings.py (hunk 2 above)
Comment (kiran-tripathy) [reply common_utilities/path_settings.py]: Yeah it isn't -- [cb4e04943a84007ae233d1c08327345a3c4afa1b](https://github.com/dimagi/dimagi-qa/pull/145/commits/cb4e04943a84007ae233d1c08327345a3c4afa1b)

File: common_utilities/generate_output_path.py (hunk 1 above)
Comment (kiran-tripathy) [reply common_utilities/generate_output_path.py]: updated : https://github.com/dimagi/dimagi-qa/pull/145/commits/8746ae0ca107fe7dae22690d90b0d8b6c3c49545

File: AppSprintPerformanceTests/CICT/CaptureReadings/conftest.py [hunk 3]
```diff
@@ -55,6 +56,7 @@ def settings(environment_settings):
             )
         return settings
     path = Path(__file__).parent.parent / "settings.cfg"
+    print(path)
```
[the remark this answers is not in the document]
Comment (kiran-tripathy) [reply AppSprintPerformanceTests/CICT/CaptureReadings/conftest.py]: updated: https://github.com/dimagi/dimagi-qa/pull/145/commits/8746ae0ca107fe7dae22690d90b0d8b6c3c49545

**Review by kbo001 — APPROVED** [review]
loking good. Approving!

**Pull request merged**